CGPs/0023.md

```diff
@@ -10,43 +10,42 @@
 
 This governance proposal enables oracle addresses to make them able to report the value of CELO/EUR for the cEUR contract, deployed on [CR3](https://github.com/celo-org/celo-proposals/blob/master/CGPs/0022.md). It also proposed to send 100 CELO to each of the oracles to cover gas fees for the first couple of years and freezing StableTokenEUR (cEUR) and ExchangeEUR.
 
-Following [CR3](https://github.com/celo-org/celo-proposals/blob/master/CGPs/0022.md), cEUR will be in an status where it would be imposible to mint or burn due empty buckets that are not possible to update (assuming no oracle reports, but cEUR will be possible to move. Freezing is meant to prevent cEUR from moving before enabling the stability mechanism.
+Following [CR3](https://github.com/celo-org/celo-proposals/blob/master/CGPs/0022.md), cEUR will be in an status where it would be imposible to mint or burn due empty buckets that are not possible to update (because Mento buckets are zero and there are no oracles enable to updated), but cEUR will be possible to move. Freezing is meant to prevent cEUR from moving before enabling the stability mechanism.
```

Comment:
```suggestion
Following [CR3](https://github.com/celo-org/celo-proposals/blob/master/CGPs/0022.md), cEUR will be in a status where it will be imposible to mint or burn due to empty buckets that are not possible to update (because Mento buckets are zero and there are no oracles enabled to update them), but cEUR will be transferable. Freezing is meant to prevent cEUR being transferred before enabling the stability mechanism.
```